non-free.md

```diff
@@ -106,3 +107,8 @@
   
 ## Ticketing
   * [SupportPal](https://www.supportpal.com/) `⊘ Proprietary` - Powerful help desk software - easy, fast and intuitive. ([Demo](http://demo.supportpal.com/)) `PHP`
+
+
+## Remote Support
+
+ * [ScreenConnect](https://www.screenconnect.com/) `⊘ Proprietary` - ScreenConnect offers lightning-fast remote support and remote access to connect instantly and solve problems faster.
```

Comment:
Is it a third party service? Or is is something you can install on your server? If not, this can't be added (not a self-hostable network service/web application)